Package.swift

```diff
@@ -1,8 +1,18 @@
+import Foundation
 import PackageDescription
 
 let package = Package(
     name: "ReactiveSwift",
-    dependencies: [
-        .Package(url: "https://github.com/antitypical/Result.git", versions: Version(3, 2, 1)..<Version(3, .max, .max)),
-    ]
+    dependencies: {
+        var deps: [Package.Dependency] = [
+            .Package(url: "https://github.com/antitypical/Result.git", versions: Version(3, 2, 1)..<Version(3, .max, .max)),
+        ]
+        if ProcessInfo.processInfo.environment["SWIFTPM_TEST_ReactiveSwift"] == "YES" {
```

Comment:
What do you think about extracting a helper at the top level for this, say an `isSwiftPackagerManagerTest` computed property?